boltdesignsystem/bolt
# DS-333: replace onClick and onClickTarget with snake case

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-333](https://pegadigitalit.atlassian.net/browse/DS-333)

## Summary

deprecate onClick and onClickTarget in BoltActionElement in favor of snake case

## Details

Replace BoltActionElement properties onClick and onClickTarget with snake case counterparts, on_click and on_click_target

## How to test

Make sure none of the demos are broken, and confirm that any precautions put in place would prevent disaster. Also confirm I didn't miss any uses of the camel case props.

## Release notes

deprecate onClick and onClickTarget for snake case.
